v7/import_blogger/import_blogger.py

```diff
@@ -154,8 +156,11 @@ def import_item(self, item, out_folder=None):
 
         if link_path.lower().endswith('.html'):
             link_path = link_path[:-5]
+        link_path = link_path.lstrip('/')
 
-        slug = utils.slugify(link_path)
+        out_path = os.path.join(self.output_folder, out_folder, link_path)
+        link_fragments = link_path.split('/')
+        slug = utils.slugify(link_fragments[-1])
```

Comment:
Are blogger slugs unique?  For example, Could there be posts on different dates `/2016/06/auto-completion-for-custom-search` and `/2014/03/auto-completion-for-custom-search`, which would cause slug clashes.